metadata/metadata_service/api/feature.py

```diff
@@ -83,13 +86,25 @@ class FeatureOwnerAPI(Resource):
     def __init__(self) -> None:
         self.client = get_proxy_client()
 
-    @swag_from('swagger_doc/table/owner_put.yml')
-    def put(self, table_uri: str, owner: str) -> Iterable[Union[Mapping, int, None]]:
-        pass
-
-    @swag_from('swagger_doc/table/owner_delete.yml')
-    def delete(self, table_uri: str, owner: str) -> Iterable[Union[Mapping, int, None]]:
-        pass
+    @swag_from('swagger_doc/feature/owner_put.yml')
+    def put(self, feature_uri: str, owner: str) -> Iterable[Union[Mapping, int, None]]:
+        try:
+            self.client.add_resource_owner(uri=feature_uri, resource_type=ResourceType.Feature, owner=owner)
+            return {'message': f'The owner {owner} for feature_uri {feature_uri} '
+                               'was added successfully'}, HTTPStatus.OK
+        except Exception:
+            return {'message': f'The owner {owner} for feature_uri {feature_uri} was '
```

Comment:
observation: bit odd there's no logging and err details are not returned. I know there also isn't for table owners.

Reply:
+1

Reply:
I'll add some!

Reply:
done